docs/unified/novacustom/initial-deployment.md

```diff
@@ -51,21 +17,34 @@ flashrom -p internal -r dump.rom
     Intel ME version (and configuration) on the device. Since vendor firmware
     has enabled Intel Boot Guard and BIOS Guard, it is not possible to do this
     from within the operating system and external flashing of the whole flash
-    chip using a programmer like the CH341a or RTE is required.
+    chip using a programmer like the CH341a or
+    [3mdeb RTE](http://docs.dasharo.com/transparent-validation/rte/introduction/)
```

Comment:
@mkopec I think @sulewskiprzemyslaw and @Pokisiekk done dedicated documenation on how to use RTE with this laptop. We should use/link it here.

Reply:
https://docs.dasharo.com/transparent-validation/rte/v1.1.0/flashing-guide/#ns5x7x-12th-generation

Reply:
Addd